spec/mailers/portal_mailer_spec.rb

```diff
@@ -22,7 +22,7 @@
     end
 
     it 'renders the sender email' do
-      expect(mail.from).to eq(['workshop.portal@gmail.com'])
+      expect(mail.from).to eq([Rails.configuration.default_reply_to_email])
```

Comment:
Same as the top comment, the from is actually exactly the senders adress, not the reply to, so we need to revert this change.
Moreover, I think "sets" would fit better than "renders" (not your change tho)